applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -183,6 +183,16 @@ int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pr
  */
 void le_audio_conn_set(struct bt_conn *conn);
 
+/**
+ * @brief	Set periodic advertising sync
+ *
+ * @param[in]	pa_sync		Pointer to the periodic advertising sync
+ * @param[in]	broadcast_id	Broadcast ID of the periodic advertising
```

Comment:
```suggestion
 * @param	pa_sync		Pointer to the periodic advertising sync
 * @param	broadcast_id	Broadcast ID of the periodic advertising
```

If all values are `[in]`, it does not make a lot of sense to define them as such :) (unless you want to do that for all `@param` in your documentation

Reply:
This is a matter of multiple people working in the same repo and not having a clear guidline if we should use [in] or not